horovod/runner/launch.py

```diff
@@ -427,9 +427,11 @@ def parse_args():
                                help='Minimum level to log to stderr from the Horovod backend. (default: WARNING).')
     group_logging_timestamp = group_logging.add_mutually_exclusive_group()
     group_logging_timestamp.add_argument('--log-hide-timestamp', action=make_override_true_action(override_args),
-                                         help='Hide the timestamp from Horovod log messages.')
```

Comment:
I presume there is no way to rename this `--log-hide-timestamp`, or deprecate it? Better name would be `--log-without-timestamp`. Even worse: `--no-log-hide-timestamp`. Are they different to `--timestamp-output`?

Reply:
Just read your comment above... Still think they should be marked deprecated and a better named flag put in its place. With the `--timestamp-output`: That is been created independent to `--log-hide-timestamp` on purpose?

Reply:
We can rename and deprecate the old flag, I will do that.

Yes, I believe it is correct to have separate flags here, since they are serving different purposes.